osu.Game/Screens/Play/Player.cs

```diff
@@ -73,6 +73,8 @@ public abstract class Player : ScreenWithBeatmapBackground, ISamplePlaybackDisab
 
         private Bindable<bool> mouseWheelDisabled;
 
+        private bool hasBackgroundToFit=false;
```

Comment:
This won't pass code style inspections. Please compare how code is formatted everywhere else and follow that example. The InspectCode tool - mentioned in the contributing guidelines - is intended to help with keeping a consistent style, so please make sure to run it.